### ISSUE TITLE
Get rid of compile/build warnings

### ISSUE THREAD

**Issue by JoshuaSBrown** [MikeHeiber/Excimontec#27]
```src/OSC_Sim.cpp:1374:0: warning: ignoring #pragma loop  [-Wunknown-pragmas]
    #pragma loop(hint_parallel(2))
 ^
src/OSC_Sim.cpp:1375:0: warning: ignoring #pragma loop  [-Wunknown-pragmas]
    #pragma loop(ivdep)
 ^
 
In file included from KMC_Lattice/src/Simulation.h:12:0,
                 from src/OSC_Sim.h:9,
                 from src/OSC_Sim.cpp:6:
KMC_Lattice/src/Event.h: In member function ‘void Excimontec::OSC_Sim::calculateExcitonEvents(Excimontec::Exciton*)’:
KMC_Lattice/src/Event.h:24:7: warning: ‘event_ptr_target’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 class Event{
       ^
src/OSC_Sim.cpp:719:9: note: ‘event_ptr_target’ was declared here
  Event* event_ptr_target;
         ^
In file included from KMC_Lattice/src/Simulation.h:12:0,
                 from src/OSC_Sim.h:9,
                 from src/OSC_Sim.cpp:6:
KMC_Lattice/src/Event.h: In member function ‘void Excimontec::OSC_Sim::calculatePolaronEvents(Excimontec::Polaron*)’:
KMC_Lattice/src/Event.h:24:7: warning: ‘event_ptr_target’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 class Event{
       ^
src/OSC_Sim.cpp:949:9: note: ‘event_ptr_target’ was declared here
  Event* event_ptr_target;
         ^
```

**Comment by JoshuaSBrown** [comment]
Addressed here: https://github.com/MikeHeiber/Excimontec/pull/35

**Comment by MikeHeiber** [comment]
Resolved in #54
